conf/ocsci/deploy_arbiter.yaml

```diff
@@ -0,0 +1,7 @@
+---
+DEPLOYMENT:
+  arbiter_deployment: True
+  ocs_operator_nodes_to_label: 4
```

Comment:
How do I define which is the arbiter zone?

Reply:
It will be determined automatically.

Reply:
This won't work for testing purposes.

Reply:
Can you please explain why? I have tested it and it worked. BTW arbiter setups have some special requirements.

Reply:
We are doing some autodetection in the UI

Reply:
- From OCS Operator perspective, there is no autodetection. StorageCluster CR have to specify all the details (as you noted, some autodetection is happening only in UI).
- We want to be in full control of the deployment process, as we do for other CI deployment configurations, which eg. specifies all the zones where we care about zones.
- Autodetection in ocs-ci would be error prone and could invalidate our testing if eg. we deploy it in a wrong environment. We would be debugging ocs-ci code instead of a product.

Reply:
Hi @mbukatov I have introduced functionality to feed zones via yaml conf files. Pls take a look